src/libpmemset/pmemset.c

```diff
@@ -297,24 +324,26 @@ pmemset_part_map(struct pmemset_part **part, struct pmemset_extras *extra,
 
 	pmemset_set_persisting_fn(set, part_map);
```

Comment:
In case of 'part_coalescing && prev_pmap' part_map is not set, and set persisting will fail.